src/builder.rs

```diff
@@ -2,8 +2,8 @@ use std::collections::HashMap;
 use std::rc::Rc;
 use std::cell::RefCell;
 
-use pwasm_std::hash::{H256, Address};
-use uint::U256;
+use pwasm_std::types::{H160, H256, U256};
+type Address = H160;
```

Comment:
Out of curiosity, why isn't `Address` exported from `pwasm-std` (or `pwasm-ethereum`)?

Reply:
the same question 🙂. I see a trade-off here also. If we would declare it everywhere that's a bit better for explicitness. But still I'd prefer to declare the alias once somewhere (in `pwasm-std` or `pwasm-abi`)..

Reply:
This is due to the design decision I have made in version `0.11` of `pwasm-std`.
It made `H160` and `Address` both distinct types which I now regret.
This is going to be reverted in an upcoming version `0.12` of `pwasm-std` because it broke way too much code.